specs/ProgrammaticSaveAs.md

```diff
@@ -0,0 +1,429 @@
+Programmatic Save As API
+===
+
+# Background
+
+The context menu has the "Save as" item to manually save the html page, image,
+pdf, or other content through a save as dialog. We provide more flexiable ways 
+to do the save as programmatically in WebView2. You can bring up the default 
+save as dialog easily. And you will be able to block default dialog, save the 
```

Comment:
"With the new API you will be able to:
- Launch the default save as dialog
- Block the default save as dialog
- Save the content silently by providing the path and save as kind
- Build your own save as UI"